docs/index.asciidoc

```diff
@@ -229,7 +229,8 @@ The retention policy to use
 
 An array containing the names of fields to send to Influxdb as tags instead 
 of fields. Influxdb 0.9 convention is that values that do not change every
-request should be considered metadata and given as tags.
+request should be considered metadata and given as tags. Tags are only send when
```

Comment:
@jklare    Nice addition to the documentation.  One minor correction:  Please change `send` to `sent` to become `Tags are only sent when present..."